src/content/docs/apis/nerdgraph/examples/nerdgraph-api-nrql-condition-alerts.mdx

```diff
@@ -83,16 +83,16 @@ mutation {
 }
 ```
 
-## NRQL baseline condition [#baseline-condition]
+## NRQL anomaly condition [#anomaly-condition]
 
-Here's an example of creating a baseline condition:
+Here's an example of creating a anomaly condition:
 
 ```
 mutation {
-  alertsNrqlConditionBaselineCreate(accountId: <var>YOUR_ACCOUNT_ID</var>, policyId: <var>YOUR_POLICY_ID</var>, condition: {
-    name: "Baseline Condition"
+  alertsNrqlConditionAnomalyCreate(accountId: <var>YOUR_ACCOUNT_ID</var>, policyId: <var>YOUR_POLICY_ID</var>, condition: {
```

Comment:
Just wanted to double-check that the condition name has actually been changed in the underlying code. Sometimes the product team cahnges the public-facing language without changing the underlying attribute/event names.

Reply:
@homelessbirds underlying code was not changed